### PR TITLE
DOCS-3365 RTF December Release

### PR DESCRIPTION
**Pull request by sleslie23**
Hi All! This PR includes the following changes:

- Adding topic/images for app versioning.
- Adding changes related to activation token.
- Revising installation workflow to reflect activation token.

### CODE REVIEW

**Review by valkyrie69 — APPROVED** [review]

File: modules/ROOT/pages/install-aws.adoc [hunk 1]
```diff
@@ -62,11 +62,7 @@ Some of the values for the variables below are available on the Create Runtime F
...
 | `key_pair` | The name of the keypair in the AWS region you are deploying to. | `my-keypair`
 | `enable_public_ips` | Provision public IP addresses for each VM. If left as false, ensure you've consulted with your network administrator on how to obtain shell/SSH access to VMs. | `false`
-| `anypoint_org_id` | The ID for your Anypoint organization, available on the Create Runtime Fabric page. | `2f99d26-a3c2-4023-b303-5528sdd5v022`
-| `anypoint_token` | An active Anypoint authentication token, available on the Create Runtime Fabric page. | `bearer 626d7810-0730-498b-bd48-544490c2226b`
-| `anypoint_region` | The Anypoint broker region you're registering to, available on the Create Runtime Fabric page. | `us-east-1`
-| `anypoint_endpoint` | The Anypoint control plane you're registering to, available on the Create Runtime Fabric page. | `anypoint.mulesoft.com`
-| `installer_url` | The URL of the Runtime Fabric installation package, available on the Create Runtime Fabric page. | `runtime-fabric.s3.amazonaws.com/...`
+| `activation_data` | Specifies the encrypted Runtime Fabric activation data | NzdlMzU1YTktMzAxMC00OGE0LWJlMGQtMDdxxxx
```
Comment (yianL) [inline modules/ROOT/pages/install-aws.adoc]: "encoded" instead of "encrypted"

File: modules/ROOT/pages/install-azure.adoc [hunk 2]
```diff
@@ -87,15 +87,11 @@ You'll need a private and public key as a PEM file to provision the VMs. This is
...
-| `Anypoint Organization ID` | The ID for your Anypoint organization, available on the Create Runtime Fabric page. | `2f99d26-a3c2-4023-b303-5528sdd5v022`
-| `Anypoint Control Plane Host` | The Anypoint control plane you're registering to, available on the Create Runtime Fabric page. | `anypoint.mulesoft.com`
-| `Installer URL` | The URL of the Runtime Fabric installation package, available on the Create Runtime Fabric page. | `runtime-fabric.s3.amazonaws.com/...`
 | `SSH User Name` | The user name to use when establishing a SSH connection to each VM. | `rtf-user`
 | `Runtime Fabric Name` | The name for your Runtime Fabric cluster. | `rtf-azure`
 | `Runtime Fabric Token` | The token used to join each other VM to the installer VM. This should be a unique value to avoid conflicts during installation. | `my-rtf-token`
 | `Controller Instance Type` | The Azure machine type to provision for each controller VM. Default set to 2 cores and 8 GiB memory. | `Standard_D2s_v3`
+| `Anypoint Activation Data` | Specifies the encrypted Runtime Fabric activation data | NzdlMzU1YTktMzAxMC00OGE0LWJlMGQtMDdxxxx
```
Comment (yianL) [inline modules/ROOT/pages/install-azure.adoc]: "encoded" instead of "encrypted"

File: modules/ROOT/pages/install-manual.adoc [hunk 3]
```diff
@@ -62,12 +62,9 @@ To begin, we will run a configuration tool. This aids in generating the environm
...
-| `RTF_ETCD_DEVICE` |  Block device path for the dedicated disk used for etcd, supporting 3000 provisioned IOPS. The minimum value required is 60 GiB. | `/dev/xvdb`
-| `RTF_AUTH_TOKEN` | An active Anypoint authentication token, available on the Create Runtime Fabric page. | `Bearer 626d7810-0730-498b-bd48-544490c2226b`
-| `RTF_REGION` | The Anypoint broker region you're registering to, available on the Create Runtime Fabric page. | `us-east-1`
-| `RTF_ORG_ID` | The ID for your Anypoint organization, available on the Create Runtime Fabric page. | `2f99d26-a3c2-4023-b303-5528sdd5v022`
-| `RTF_ENDPOINT` | The Anypoint control plane you're registering to, available on the Create Runtime Fabric page. | `anypoint.mulesoft.com`
+| `RTF_DOCKER_DEVICE` |  Block device path for the dedicated disk used for Docker, supporting 1000 provisioned IOPS. | `/dev/xvdc`
+| `RTF_ETCD_DEVICE` |  Block device path for the dedicated disk used for etcd, supporting 3000 provisioned IOPS. | `/dev/xvdb`
+| `RTF_ACTIVATION_DATA` | Specifies the encrypted Runtime Fabric activation data | NzdlMzU1YTktMzAxMC00OGE0LWJlMGQtMDdxxxx
```
Comment (yianL) [inline modules/ROOT/pages/install-manual.adoc]: "encoded" instead of "encrypted"

File: modules/ROOT/pages/install-manual.adoc [hunk 4]
```diff
@@ -62,12 +62,9 @@ To begin, we will run a configuration tool. This aids in generating the environm
...
 | `RTF_INSTALL_PACKAGE_URL` | The URL of the Runtime Fabric installation package, available on the Create Runtime Fabric page. _If this is not specified, the installer will expect you to manually copy the installation package to_ `/opt/anypoint/runtimefabric/installer.tar.gz`. | `https://s3.amazonaws.com/runtimefabric/...`
```
Comment (yianL) [inline modules/ROOT/pages/install-manual.adoc]: remove parameter `RTF_INSTALL_PACKAGE_URL` as well

**Review by sundippatel — APPROVED** [review]
Made changes to streamline installation pages and conform to the new activation flow.

**Review by mcocaro — APPROVED** [review]

**Pull request merged**